### PR TITLE
[RW-10717][risk=no]  Disable country updates on researcher profile page

### PR DESCRIPTION
**Pull request by NehaBroad**
https://github.com/all-of-us/workbench/assets/34481816/dcabc47b-6a2b-4a47-97e0-17429cf1235b

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: ui/src/app/pages/profile/profile-component.tsx [hunk 1]
```diff
@@ -355,11 +347,21 @@ export const ProfileComponent = fp.flow(
                 content='This field cannot be edited'
                 disabled={!props.disabled}
               >
-                <TextInput
-                  data-test-id={props.id || valueKey}
-                  {...inputProps}
-                  style={{ ...styles.inputStyle, ...inputProps.style }}
-                />
+                <div>
```
Comment (NehaBroad) [inline ui/src/app/pages/profile/profile-component.tsx:350]: Fix existing bug: tooltip was not working for disabled inputs

File: ui/src/app/pages/profile/profile-component.tsx [hunk 2]
```diff
@@ -543,9 +545,22 @@ export const ProfileComponent = fp.flow(
...
                     })}
                     {makeProfileInput({
-                      title: 'Country',
+                      title: (
+                        <FlexRow style={{ gap: '10px' }}>
+                          <label>Country</label>
+                          <TooltipTrigger
+                            side={'right'}
+                            content={
+                              'If you need to update your country, email drcsupport@researchallofus.org with your updated country.'
```
Comment (NehaBroad) [inline ui/src/app/pages/profile/profile-component.tsx:554]: only issue with this is that user CANNOT copy the email id

File: ui/src/app/pages/profile/profile-styles.ts [hunk 3]
```diff
@@ -21,6 +21,9 @@ export const styles = reactStyles({
     fontSize: 14,
     marginRight: 20,
   },
+  disabledInput: {
+    backgroundColor: colorWithWhiteness(colors.disabled, 0.6),
```
Comment (NehaBroad) [inline ui/src/app/pages/profile/profile-styles.ts:25]: right now , the disabled text boxes are were very hard to distinguish added a little more grey to make it obvious

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java [hunk 4]
```diff
@@ -852,6 +852,18 @@ public void updateGivenName_badRequest() {
         });
   }
 
+  @Test
+  public void updateProfile_badRequest_updateCountry() {
```
Comment (Peter-Lavigne) [inline api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:856]: Thank you for writing backend validations and a straightforward test!

**Review by Peter-Lavigne — APPROVED** [review]
@NehaBroad this PR is perfect. It resolves the ticket, was easy to review, validates both the UI and API, and is tested both simply and thoroughly. Thank you!

**Pull request merged**